lib/oauth.js

```diff
@@ -74,16 +75,19 @@ module.exports = new Model({
       console.info('Setting app ID to', appID);
       ls.get(LOCAL_STORAGE_PREFIX + 'clientAppId', appID);
       this._clientAppId = appID;
+      if (options.customRedirects) this._customRedirects = options.customRedirects;
```

Comment:
Isn't there a chance that `options.customRedirects` might throw a `property X of undefined` error?

Since we're introducing the `options` argument in this PR, it might be a good idea to check if `options` exists, or use a default value.

Reply:
Yep, it should have a default. 👍